docs/index.asciidoc

```diff
@@ -414,6 +415,16 @@ instructions into the query.
 If enabled, SSL will be used when communicating with the Elasticsearch
 server (i.e. HTTPS will be used instead of plain HTTP).
 
+[id="plugins-{type}s-{plugin}-ssl_certificate_verification"]
+===== `ssl_certificate_verification`
+
+* Value type is <<boolean,boolean>>
+* Default value is `true`
+
+Option to validate the server's certificate. Disabling this severely compromises security.
```

Comment:
Although our docs call out _"Disabling this severely compromises security"_ and links to a 12-page white-paper that is tangentially-related to improper certificate validation, I think it may be worth calling out at least a few of the abuse scenarios in the doc here:

```suggestion
Option to validate the server's certificate. Disabling this severely compromises security.
When certificate validation is disabled this plugin implicitly trusts the machine
resolved at the given address without validating its proof-of-identity, which can
lead to this plugin transmitting credentials to or processing data from an untrustworthy
man-in-the-middle or other compromised infrastructure.
```